vspace/src/main/java/edu/asu/diging/vspace/core/factory/impl/LocalizedTextFormCreation.java

```diff
@@ -0,0 +1,38 @@
+package edu.asu.diging.vspace.core.factory.impl;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+import org.thymeleaf.util.StringUtils;
+
+import edu.asu.diging.vspace.core.model.IExhibitionLanguage;
+import edu.asu.diging.vspace.core.model.ILocalizedText;
+import edu.asu.diging.vspace.web.staff.forms.LocalizedTextForm;
+
+@Service
+public class LocalizedTextFormCreation {
```

Comment:
creation is an event not an "actor" (meaning a creation is the outcome of a creator creating something). Services are the classes that do something, not the output of action.